pallets/proofs/src/tests/mod.rs

```diff
@@ -1,6 +1,43 @@
+use core::str::FromStr;
+
+use cid::Cid;
+use codec::Decode;
+use polka_storage_proofs::{Bls12, Proof};
+use primitives::{MAX_POST_PROOF_BYTES, MAX_PROOFS_PER_BLOCK};
+use sp_core::ConstU32;
+use sp_runtime::BoundedVec;
+
 mod porep;
 mod post;
 
 pub const TEST_SEED: [u8; 16] = [
     0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06, 0xbc, 0xe5,
 ];
+
+pub fn raw_commitment_from_hex(hex_str: &str) -> [u8; 32] {
+    Cid::from_str(hex_str)
+        .unwrap()
+        .hash()
+        .digest()
+        .try_into()
+        .unwrap()
+}
+
+/// Loads a proof file generated by polka-storage-provider-client either with porep or post command.
+pub fn load_proof_file(
+    path: &str,
+) -> BoundedVec<BoundedVec<u8, ConstU32<MAX_POST_PROOF_BYTES>>, ConstU32<MAX_PROOFS_PER_BLOCK>> {
+    let proof_bytes = std::fs::read(path).unwrap().to_vec();
```

Comment:
```suggestion
    let proof_bytes = std::fs::read(path).unwrap();
```
read() already returns an `Vec<u8>`.